.gitignore

```diff
@@ -15,3 +15,6 @@ public/app.js
  
 # Misc
 db/seeding/*.json 
+
+# Vim
+*.swp
```

Comment:
After editing files in vim, you can delete them lol. But since you're increasing vim usage, I think this is also okay to do.